sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -292,6 +417,22 @@ public void processElement(@SuppressWarnings("unused") ProcessContext c) {
   }
 
   @Test
+  public void testProcessElementExceptionWithReturn() throws Exception {
```

Comment:
Do we have test that use an input of type `long[]` (just for sanity checking things)